sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -425,7 +425,7 @@ object SQLConf {
     .createWithDefault(true)
 
   val ADAPTIVE_EXECUTION_SKEWED_JOIN_ENABLED =
-    buildConf("spark.sql.adaptive.optimizeSkewedJoin.enabled")
+    buildConf("spark.sql.adaptive.skewedJoinOptimization.enabled")
```

Comment:
ah I missed this one. We should update other skew join related configs to use the `skewedJoinOptimization` prefix.

@JkSelf can you help to fix it?

Reply:
fix it in [PR#27630](https://github.com/apache/spark/pull/27630).